.github/workflows/checks.yml

```diff
@@ -91,4 +91,5 @@ jobs:
 
       - name: Validate URLs
         run: |
-          awesome_bot ./*.md src/*.sol test/**/*.sol test/public/**/*.sol --allow-dupe --request-delay 0.4
+          awesome_bot ./*.md src/*.sol test/**/*.sol test/public/**/*.sol --allow-dupe --request-delay 0.4 \
+          --white-list https://github.com/pcaversaccio/createx/issues/new?assignees=pcaversaccio&labels=new+deployment+%E2%9E%95&projects=&template=deployment_request.yml&title=%5BNew-Deployment-Request%5D%3A+
```

Comment:
So the reason why I put it on the `--white-list` is that if `awesome_bot` validates the URL, the bot is of course not signed in on GitHub and GitHub will make a redirect, which I disallow in the CI check to always have the correct link. You can see this behaviour [here](https://github.com/pcaversaccio/createx/actions/runs/6748606751/job/18347284380#step:5:64). I resolve this convo since it's intended behaviour.

Reply:
Actually had to add `""` to the link since otherwise it didn't get recognised: https://github.com/pcaversaccio/createx/pull/26/commits/e21dd2a64ca516038407e4edf03097b1a2bdb75a